jar.go

```diff
@@ -396,6 +396,34 @@ func (j *Jar) deleteExpired(now time.Time) {
 	}
 }
 
+// RemoveAll removes any cookies from the jar that were set for the given host.
+func (j *Jar) RemoveAll(host string) {
```

Comment:
Maybe name this RemoveAllHost so that we can still provide RemoveAll(url) later?